docs/study-def-dates.md

```diff
@@ -71,6 +78,8 @@ actions:
         cohort: output/input-2020-01-01.csv
 ```
 
+This can also be used to define a range of dates over which to run the study_definition, usually when working with [Measures](https://docs.opensafely.org/en/latest/measures/).
```

Comment:
I think the target of this link should be a local relative one `(measures.md)`, rather than the absolute url `(https://docs.opensafely.org/en/latest/measures/)`

Reply:
Thanks, good suggestion - I've now done this, hopefully correctly!